lib/source_file.h

```diff
@@ -126,6 +127,14 @@ class SourceInfo final {
     int line = -1;
     int column = -1;
     cstring srcBrief = ""_cs;
+
+    // add a comment to 'before' or 'after' array of comments
```

Comment:
This comment is not very useful, it does not state anything the type does not. Either delete it, or expand on documentation details (and make it Doxygen-style.).